staff/adrian-zhu/precourse/Airflight/index.js

```diff
@@ -0,0 +1,231 @@
+function airlines(){
+    let userName=askUserName();
+    alert(`Encantad@ ${userName}, \n¡Bienvenido a Skylab Airlines!!`);
+    console.log(`<--------------------------------------------------------------->`);    
+    showFlights(flights);
+    console.log(`<--------------------------------------------------------------->`);
+    let avg=average();
+    console.log(`El coste promedio de nuestros vuelos ${avg}€  `);
+    console.log(`<--------------------------------------------------------------->`);
+    withStop();
+    console.log(`<--------------------------------------------------------------->`);
+    console.log('Los últimos vuelos:');
+    console.log(`<--------------------------------------------------------------->`);
+    lastFlights();
+    let prof=askProfileUser();
+}
+
+function withStop(){
+    
+    for(let i=0;i<flights.length; i++){
+
+        if(flights[i].scale==true){
+        
+        console.log(`Los siguientes vuelos tienen escalas--> Vuelo ${flights[i].id} desde ${flights[i].from} hasta ${flights[i].to} `);
+        }
+        
+    }
+    return
+}
+
+function average(){
+    let sum=0;
+    for(let i=0;i<flights.length; i++){
+        
+        sum+= flights[i].cost;
+    }
+    return Math.floor(sum/flights.length);
+}
+
+function showFlights(flightList){
+
+    for(let i=0;i<flightList.length; i++){
+        let scale='';
+        if(flightList[i].scale==true){
+        scale='Tiene stops'
+        }else{
+        scale="No tienen stops.";    
+        }
+        console.log(`Los vuelos id ${flightList[i].id} desde ${flightList[i].from} hasta ${flightList[i].to} costes ${flightList[i].cost}€ and ${scale}`);
+    }
+    return
+}
+function askUserName(){
+        let count=0;
+        var name=prompt('El nombre de usted?');
+        for (let i=0; i<name.length;i++){
+            count+= 1;
+            if (name[i]==" "){
+            break;
+            }
+        }
+        let firstname=name.slice(0,count);
+        return firstname;
+
+}
+function lastFlights(){
+
+    let lastFlights=flights.slice(5,10);
+
+    for(let i=0;i<lastFlights.length; i++){
+        console.log(`To ${lastFlights[i].to}`);
+    }
+    return
+}
+function askProfileUser(){
+
+    let profile=prompt('Es su perfil administrador o usuario?');
+    if(profile=='ADMINISTRADOR'|| profile=='administrador' || profile=='Administrador'){
```

Comment:
Could have done .toLowerCase() and avoid having to write all the different options (||).

Reply:
Thanks, it is fixed